lib/preneeds/service.rb

```diff
@@ -1,50 +1,74 @@
 # frozen_string_literal: true
 require 'common/client/base'
+require 'common/client/concerns/streaming_client'
+require 'net/http/post/multipart'
 
 module Preneeds
   class Service < Common::Client::Base
+    include Common::Client::StreamingClient
```

Comment:
yea, you're right -- i was experimenting with something and forgot to remove this.